getting_started.md

```diff
@@ -52,25 +52,41 @@ using Flux
 ```
 <br>
 
-### Step 2: Create some train data
-For this example, create some random train data `x` and `y` arrays:
+### Step 2: Create the training data
+First, we'll write a function that generates our "true" data. We'll use to use Flux to recover `W_truth` and `b_truth` by looking only at examples of the `ground_truth` function.
 
 ```julia
-x = rand(5)
-y = rand(2) 
+W_truth = [1 2 3 4 5;
+            5 4 3 2 1]
+b_truth = [- 1.0; -2.0]
+ground_truth(x) = W_truth * x .+ b_truth
```

Comment:
```suggestion
ground_truth(x) = W_truth*x .+ b_truth
```
To match the function below

Reply:
Actually, I just totally forgot about those. Thanks for the catch.